ndx-multichannel-volume/README.md

```diff
@@ -0,0 +1,53 @@
+# ndx-multichannel-volume Extension for NWB
+
+Description of the extension
+
+## Installation
+
+To install this package on Unix/macOS, run in command line
+python3 -m pip install --index-url https://pypi.org/simple/ --no-deps ndx-multichannel-volume
+
+On windwos, run 
+
+py -m pip install --index-url https://pypi.org/simple/ --no-deps ndx-multichannel-volume
```

Comment:
As far as I know, on common installations of Python on Windows, you can just call `python xx`

Reply:
I have never really coded on a Windows machine. Do you mean you can just call 'python install --index-url https://pypi.org/simple/ --no-deps ndx-multichannel-volume' or something else?

What I'm seeing online shows pretty much what I have here currently, but please let me know if there's something simpler to put here.

Reply:
I meant that you can call 
```
python -m pip install --index-url https://pypi.org/simple/ --no-deps ndx-multichannel-volume
```

but it is much more common nowadays to tell users to run 
```
python -m pip install ndx-multichannel-volume
```
regardless of OS. This will use pypi by default and install dependencies (unless there was a specific reason to have users not install deps). On Mac/Linux nowadays, `python` and `python3` are the same thing unless the user has a really old OS, so you can simplify your install directions to just the second command above.